compiler/src/iree/compiler/Dialect/LinalgExt/Transforms/ConvertConv2DToWinograd.cpp

```diff
@@ -273,15 +274,15 @@ class ConvertConvToWinograd final : public OpRewritePattern<ConvOp> {
 
     // Add BatchMatmulOp
     SmallVector<int64_t> bmmShape(collapsedShape.begin(), collapsedShape.end());
-    Value output = convOp.getOutputs()[0];
-    auto outputType = cast<RankedTensorType>(output.getType());
     SmallVector<int64_t> outputShape(outputType.getShape());
     if (isNchwFchw) {
       permute<IREE::LinalgExt::Permutation::NCHW_TO_NHWC>(outputShape);
     }
     bmmShape[2] = outputShape[3];
-    auto bmmOutputType = RankedTensorType::get(bmmShape, elementType);
-    emptyTensor = rewriter.create<tensor::EmptyOp>(loc, bmmShape, elementType);
+    auto bmmOutputType = RankedTensorType::get(bmmShape, outElemType);
+    emptyTensor = rewriter.create<tensor::EmptyOp>(loc, bmmShape, outElemType);
```

Comment:
IMO, it's better to have different variable, instead of reusing `emptyTensor`. Otherwise, people need to track the use of the variable. People could also introduce bugs when they are updating the code.